.github/workflows/knative-style.yaml

```diff
@@ -60,3 +60,38 @@ jobs:
               exit 1
           fi
           echo "${{ github.repository }} is formatted correctly."
+
+      # This is mostly copied from https://github.com/get-woke/woke-action-reviewdog/blob/main/entrypoint.sh
+      # since their action is not yet released under a stable version.
+      - name: Language
+        shell: bash
+        env:
+          REVIEWDOG_GITHUB_API_TOKEN: ${{ github.token }}
+          WOKE_VERSION: v0.1.11
+        run: |
+          set -e
+          cd "${GITHUB_WORKSPACE}" || exit 1
+
+          TEMP_PATH="$(mktemp -d)"
+          PATH="${TEMP_PATH}:$PATH"
+
+          echo '::group::🐶 Installing reviewdog ... https://github.com/reviewdog/reviewdog'
+          curl -sfL https://raw.githubusercontent.com/reviewdog/reviewdog/master/install.sh | sh -s -- -b "${TEMP_PATH}" 2>&1
+          echo '::endgroup::'
+
+          echo '::group:: Installing woke ... https://github.com/get-woke/woke'
+          curl -sfL https://raw.githubusercontent.com/get-woke/woke/main/install.sh | sh -s -- -b "${TEMP_PATH}" "${WOKE_VERSION}" 2>&1
+          echo '::endgroup::'
+
+          echo "vendor\nthird_party" > .wokeignore
+
+          echo '::group:: Running woke with reviewdog 🐶 ...'
+          woke --output simple \
+            | reviewdog -efm="%f:%l:%c: %m" \
+                -name="woke" \
+                -reporter="github-pr-check" \
+                -filter-mode="added" \
```

Comment:
IIUC, this will trigger on new & updated lines, but not on new filenames. Do we have a plan for new filenames? (Actually executing that plan need not block this PR.)

https://github.com/reviewdog/reviewdog#filter-mode

Reply:
That is true, i'll try to see if I can get this contributed to the tool directly. Seems useful for everybody in this context.